content/blog/2020-05-googlesheets4-0-2-0/index.Rmarkdown.orig

```diff
@@ -0,0 +1,186 @@
+---
+title: googlesheets4 0.2.0
+author: Jenny Bryan
+date: '2020-05-10'
+slug: googlesheets4-0-2-0
+categories:
+  - package
+tags:
+  - googlesheets4
+  - google
+photo:
+  author: Stephane YAICH
+  url: https://unsplash.com/photos/ufxd6aU0w9U
+---
+
+<!-- index.Rmarkdown is generated from index.Rmarkdown.orig -->
+<!-- Please edit that file and run precompile.R -->
+
+```{r setup, include = FALSE}
+can_decrypt <- gargle:::secret_can_decrypt("googlesheets4")
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>",
+  eval = can_decrypt
+)
+options(tibble.print_min = 5L, tibble.print_max = 5L)
+```
+
+```{r eval = !can_decrypt, echo = FALSE, comment = NA}
+message("No token available. Code chunks will not be evaluated.")
+```
+
+```{r auth, include = FALSE}
+googlesheets4:::gs4_auth_docs()
+```
+
+We're ecstatic to announce the release of googlesheets4 0.2.0 on CRAN:
+
+[googlesheets4.tidyverse.org](https://googlesheets4.tidyverse.org)
+
+googlesheets4 is a package to work with Google Sheets from R. Although version 0.1.0 debuted on CRAN in late 2019, I've waited to blog about it until I considered googlesheet4 a replacement for an older package called googlesheets. That day is here! In particular, googlesheets 0.2.0 can create and edit Sheets.
+
+Install googlesheets4 from CRAN like so:
+
+```{r eval = FALSE}
+install.packages("googlesheets4")
+```
+
+Then attach it for use via:
+
+```{r}
+library(googlesheets4)
+```
+
+googlesheets4 is already documented through [several articles](https://googlesheets4.tidyverse.org/articles/index.html) and, as always, you can find detailed notes about all changes in the [change log](https://googlesheets4.tidyverse.org/news/index.html).
```

Comment:
I think the goal of this paragraph should be to:

* emphasise utility of existing website
* point interested user to full changelog
* briefly preview the rest of the post

So the "already documented" strikes a bit of a weird note, and I think you need a segue into the rest of the post.